DEVELOPERS.md

```diff
@@ -208,5 +207,25 @@ Here is a template to help get started:</br>
 }
 ```
 
-## `tsconfig.packages.json`
+### `tsconfig.packages.json`
 Each package should also be listed in the root level `tsconfig.packages.json` as well. 
+
+## New Coin Implementation
+
+SDK coins provide a modular approach to a monolithic architecture. This and all BitGoJS SDK coins allow developers to use only the coins needed for a given project.
+
+To implement a new coin for BitGoJS, from the root BitGoJS/ run:
+
+```bash
+npm install -g yo
```

Comment:
do we need a global install for this? Can we just use the one installed in the monorepo root with `npx` ?

Reply:
Are you referring to `yeoman-generator`?

Reply:
We could add it as a dependency. Most use cases in general is a global install to use yeoman. I figured most users wouldn't be making use of it so only users that actually need it could globally install it.

Reply:
okay, sounds reasonable.